.circle/deployment

```diff
@@ -9,17 +9,29 @@ git clone https://${MACHINE_USER}:${MACHINE_PASSWORD}@github.com/StackStorm-Exch
 
 # Move the pack.yaml file to the index
 cp pack.yaml ~/index/v1/packs/"${PACK_NAME}".yaml
+
 # Rebuild the index JSON
 ~/virtualenv/bin/python ~/ci/.circle/index.py ~/index/v1/
+
+# Check if an icon has been added or changed
+ICON_TARGET="~/index/v1/icons/${PACK_NAME}.png"
+if [ -f icon.png ] && { [ ! -f ${ICON_TARGET} ] || ! cmp -s icon.png ${ICON_TARGET}; }
+then
+  echo "Uploading the pack icon..."
+  mkdir -p ~/index/v1/icons/
+  cp icon.png ${ICON_TARGET}
```

Comment:
Yep! I'm also considering resizing them, but I have no idea what size I'll need for the UI yet; it's not final.